src/utilities/useI18n.ts

```diff
@@ -1,10 +1,137 @@
+      scanErrorDescription: 'We were unable to connect Zone.',
+      universal: {
+        copySaveTokenTitle: 'Copy and save token',
+        copySaveTokenAlertMessage: 'You won’t see the token again! Make sure to save it somewhere now.',
+        connectZoneTitle: 'Connect Zone',
+        connectZoneDescription1: 'Copy and paste the following configuration into the config.yaml on your local machine.',
+        connectZoneFileName: 'config.yaml',
+        connectZoneConfig: `environment: universal
+mode: zone
+multizone:
+  zone:
+    name: {zoneName}
+    globalAddress: {globalKdsAddress}
+kuma:
+  multizone:
+    kds:
+      auth:
+        cpTokenInline: {token}`,
+        connectZoneCommand: 'kuma-cp run -c config.yaml',
+        connectZoneDescription2: 'Next, run the following command to connect the Zone.',
+      },
+      kubernetes: {
+        copySaveTokenTitle: 'Copy and save token',
+        copySaveTokenAlertMessage: 'You won’t see the token again! Make sure to save it somewhere now.',
+        prerequisitesTitle: 'Prerequisites',
+        prerequisitesStep1Label: 'Kubernetes cluster',
+        prerequisitesStep1LabelAddendum: 'with load balancer',
+        prerequisitesStep1Description: 'Konnect is compatible with all distributions of Kubernetes, including Minikube, AKS, EKS, GKE and OpenShift.',
+        prerequisitesStep2Label: 'kubectl or oc access',
+        prerequisitesStep2Description: 'You have kubectl or oc (if working with OpenShift) installed and configured to communicate to your Kubernetes TLS.',
+        prerequisitesStep3LinkTitle: 'Helm 3',
+        prerequisitesStep3Tail: 'is installed.',
+        helmTitle: 'Set-up Helm',
+        helmDescription: 'On your local machine, create a namespace in your Kubernetes cluster and pull down the kong Helm repo.',
+        helmStep1Description: 'Create a namespace',
+        helmStep1Command: 'kubectl create namespace kuma-system',
+        helmStep2Description: 'Add the Kong charts repository',
+        helmStep2Command: 'helm repo add kuma https://kumahq.github.io/charts',
+        helmStep3Description: 'Update Helm',
+        helmStep3Command: 'helm repo update',
+        secretTitle: 'Set-up secret',
+        secretDescription1: 'Create a secret.',
+        secretCommand: `echo "
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cp-token
+  namespace: kuma-system
+type: Opaque
+data:
+  token: {token}
+" | kubectl apply -f -`,
+        connectZoneTitle: 'Connect Zone',
+        connectZoneDescription1: 'Copy and paste the following configuration into the Helm chart values.yaml on your local machine.',
+        connectZoneFileName: 'values.yaml',
+        connectZoneConfig: `controlPlane:
+  mode: zone
+  zone: {zoneName}
+  kdsGlobalAddress: {globalKdsAddress}
+  secrets:
+    - Env: KUMA_MULTIZONE_ZONE_KDS_AUTH_CP_TOKEN_INLINE
+      Secret: cp-token
+      Key: token
+ingress:
+  enabled: {zoneIngressEnabled}
+egress:
+  enabled: {zoneEgressEnabled}`,
```

Comment:
**Comment**: A downside of putting config templates like this into i18n is that you can’t do more fine-grained JavaScript logic to _add_ `ingress.enabled` only when the setting is enabled, you can just flip the boolean.